pkgs/development/python-modules/netbox-reorder-rack/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  netbox,
+  python,
+}:
+
+buildPythonPackage rec {
+  pname = "netbox-reorder-rack";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "minitriga";
+    repo = "netbox-reorder-rack";
+    rev = "v${version}";
+    hash = "sha256-UbTq3DHxSczDxQIUyAHCVJG3SZdqgyS9ULNvuDKQvEY=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  checkInputs = [ netbox ];
+
+  preFixup = ''
+    export PYTHONPATH=${netbox}/opt/netbox/netbox:$PYTHONPATH
+  '';
+
+  dontUsePythonImportsCheck = python.pythonVersion != netbox.pythonVersion;
```

```suggestion
  dontUsePythonImportsCheck = python.pythonVersion != netbox.python.pythonVersion;
```

Why not build netbox with the correct python version?

---

NetBox itself is a python "application", so it's only built for one version of Python, in my opinion it doesn't make much sense to build it multiple times.